src/components-barnes-toolkit/scss/setup/_variables.scss

```diff
@@ -108,5 +126,15 @@ $headerHeightsShopify: (
   large: 110px,
   medium: 110px,
   small: 70px,
-  xsmall: 70px
+  xsmall: 70px,
+);
+
+// Centennial Logo Padding
+$logo100Padding: (
+  xxlarge: 60px,
+  xlarge: 60px,
+  large: 50px,
+  medium: 45px,
```

Comment:
I think all of these are a bit too high and can be halved:
```
xxlarge: 30px
xlarge: 30px,
large: 25px,
```
And I don't think we need to include medium and lower since they will be the horizontal versions!